keps/sig-auth/20190730-oidc-discovery.md

```diff
@@ -195,10 +239,66 @@ configuration:
 ```
 
 The API server would treat these as `nonResourceURLs`, and restrict access
-appropriately. We will consider expanding `system:public-info-viewer` RBAC
-ClusterRole to grant access to the new paths; some other roles may have
-permission already via a pattern match on `nonResourceURLs` (e.g.
-`cluster-admin`).
+appropriately. We will not initially write default RBAC rules for these
```

Comment:
A default clusterrole that includes the discovery and jwk nonResourceURLs would make sense to create, but leave it up to the cluster administrator to bind that role to specific users, or all authenticated users, or all unauthenticated users.

Reply:
That's true. Updated the PR and this KEP.